docker/alpine/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.12-alpine3.10 as builder
+
+RUN apk add --no-cache git gcc g++ ca-certificates \
+  && go get -u github.com/drakkan/sftpgo
+
```

Comment:
try to embed the build info inside the executable and allow to get a stable version too, for example
```
RUN go get -d github.com/drakkan/sftpgo
WORKDIR /go/src/github.com/drakkan/sftpgo
# uncomment the next line to get the latest git version
#RUN git checkout `git rev-list --tags --max-count=1` 
RUN go build -i -ldflags "-s -w -X github.com/drakkan/sftpgo/utils.commit=`git describe --always --dirty` -X github.com/drakkan/sftpgo/utils.date=`date -u +%FT%TZ`" -o sftpgo
```

Reply:
yes, it's fixed